src/Microsoft.AspNet.Mvc.Core/Formatters/MediaTypeHeaderValueComparer.cs

```diff
@@ -11,16 +11,16 @@ namespace Microsoft.AspNet.Mvc
     /// Implementation of <see cref="IComparer{T}"/> that can compare accept media type header fields
     /// based on their quality values (a.k.a q-values).
     /// </summary>
-    public class MediaTypeWithQualityHeaderValueComparer : IComparer<MediaTypeWithQualityHeaderValue>
+    public class MediaTypeHeaderValueComparer : IComparer<MediaTypeHeaderValue>
```

Comment:
yeah it's not really important just weird that we expose it as a public class on MVC